content/syllabus-written-assignments/introduction/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
+ready: true
+title: Introduction to written assignments
+---
+
+## Intro
+
+We are adding written assignments as an important part of your learning experience. These assignments are designed to help you learn better and overcome common challenges.
+
+## Why
+
+Sometimes, just reading content might not be enough to fully understand certain topics. So, we want to make sure you can apply what you've learned and express your understanding clearly.
+
+## Outcomes
+
+By completing these written assignments, you'll be more engaged with the course material, and it will be easier for you to remember what you've learned.
```

Comment:
The learners will probably ask: "What writing assignments?" .

And aren't all of these outcomes really just reasons? Why are some reasons under "why" and others are under "outcomes"?